docs/circuit-test.md

```diff
@@ -44,6 +44,49 @@ View a history of completed circuit tests, including the provider the call route
 + **Filters** pop out on the right allowing you to filter some of your data.
 + Adjust the Column ordering.
 
+|Parameter|Description|
+|---------|-----------|
+|**Created**| When the log was created mm/dd/yyyy|
+|**Provider**|List of your providers|
+|**MOS- Mean Opinion Score**|To find the call quality|
```

Comment:
Used to evaluate the perceived quality of audio.

Reply:
done